### PR TITLE
fix the "Buttons must have discernible text" error for page buttons

### PR DESCRIPTION
**Pull request by stukalin**
The library causes Google Lighthouse to complain that "Buttons must have discernible text". I've added `aria-label` attribute which fixes the issue. 

I also tried to run unit tests. I must admit they are really hard to understand and also the link `http://localhost:3000/tests/tests.html` didn't work for me: I had to open each one separately, i.e. `http://localhost:3000/test/base.html` etc. As I understood the test for those buttons is in the `customization.html` suite and that one I wasn't able to run because of some js errors.

Long story short: I updated unit tests but I couldn't run them. If you could adjust the contributing guide to the current state of things or just guess what could go wrong on my side, that'd be simply perfect!

### CODE REVIEW

**Comment by ganlanyuan** [comment]
Thanks,
